gwadoc/doc_en.py

```diff
@@ -821,14 +821,30 @@
 """
 relations.meronym.ex.en = "hand/finger"
 relations.meronym.exe.en = """
+ * *wheels* ``meronym`` *automobile*
+ * *crown* ``meronym`` *hat*
+ * *faces* ``meronym`` *people*
+ * *leaf* ``meronym`` *tree*
```

Comment:
Are these examples from the EWN documentation? If so I think we need some attribution, and can you add it to the other pull request?

Reply:
I have made changes to the examples in the new PR. Since this PR is almost a repetition of the other one. I have closed this one. You can drop your comment on the most recent PR. Attributions have been added but I have some issues making it clickable